js/visearch.js

```diff
@@ -17,6 +17,9 @@
   const STAGING_ENDPOINT = 'https://search-dev.visenze.com';
   const ANALYTICS_STAGING_ENDPOINT = 'https://staging-analytics.data.visenze.com/v3';
   const isFunction = require('lodash.isfunction');
+  const pjson = require('../package.json');
+  const SDK = 'visearch js sdk';
```

Comment:
no need to include the word "visearch"

Reply:
this is differentiate between different javascript sdk we have.
for the tracking sdk, i have set it to `tracking js sdk`

Reply:
ok